packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -162,18 +166,23 @@ export class FacetSearch {
   }
 
   private get showMoreSearchResults() {
-    if (this.props.controller.facetSearchState.values.length === 0) {
+    if (
+      this.props.controller.facetSearchState.values.length === 0 ||
+      !this.props.controller.facetSearchState.moreValuesAvailable
+    ) {
       return null;
     }
 
     return (
-      <button
-        class="px-1 text-primary"
-        onMouseDown={(e) => e.preventDefault()}
+      <li
         onClick={() => this.props.controller.facetSearch.showMoreResults()}
+        onMouseDown={(e) => e.preventDefault()}
+        part="suggestion"
+        class="suggestion cursor-pointer flex flex-row items-center px-2 text-sm text-primary"
+        value={FacetSearch.ShowMoreResults}
```

Comment:
It's strange that the value here is `1`, just to pick it up inside of onSelectValue?

Instead of adding a condition inside `onSelectValue`, could we replace the preventDefault in the  `onMouseDown` on line 179 with the call to `facetSearch.showMoreResults`?

Reply:
That would work for the mouse interaction but it wouldn't work with the keyboard pressing enter (which is actually an event on the input which triggers a search on the selected value through the combo-box class).

Reply:
Also that was a mistake it was supposed to be `-1` just to ensure it wasn't a valid index.